Project.toml

```diff
@@ -44,10 +44,10 @@ AtmosphericProfilesLibrary = "0.1.7"
 ClimaComms = "0.6.8"
 ClimaCore = "0.14.34"
 ClimaDiagnostics = "0.2.12"
-ClimaParams = "0.10.32"
+ClimaParams = "0.10.35"
```

Comment:
Is ClimaAtmos now incompatible with previous versions of these 3 packages? Since none of the atmos code was changed in this PR, I expect it's still compatible with the previous versions. If that's true, you can revert the changes in this file and just run `] up` to update the Manifest.

Reply:
If CloudMicrophysics v0.24.2 is only compatible with TD v0.12.13+ and ClimaPrams v0.10.35+, those should be specified in the compat of the CloudMicrophysics v0.24.2 Project.toml. ClimaAtmos is still compatible with any CloudMicrophysics, and it will use the right versions of the other dependencies based on the CloudMicrophysics version it's using.

So I think these need to be specified in CloudMicrophysics, but not in ClimaAtmos.

Reply:
Ok, so I should ignore my knowledge that it will not work with older versions of ClimaParams and just update the CloudMicrophysics version? I'm just worried that it will lead to errors for other people when they update ClimaParams but not CloudMicrophysics. I think we should ahve made a breaking release of CloudMicrophysics at some point to prevent it from happening

Reply:
I'll fix the toml files there